sdk/communication/AzureCommunication/Source/Authentication/CommunicationPolicyTokenCredential.swift

```diff
@@ -29,44 +29,47 @@ import AzureCore
 #endif
 import Foundation
 /**
- The Azure Communication Services authentication policy.
+ The Azure Communication Services token credential used to authenticate pipeline requests.
```

Comment:
Should there be any diff here? I should think what is in master would trump this since you are only working on Chat.

Reply:
There's some diff expected as Josh committed changes regarding the token credential to the feature branch. Added @JoshuaLai as reviewer to make sure the merge is as expected